assets/src/scss/layout/_country-selector.scss

```diff
@@ -0,0 +1,200 @@
+
+#country-selector {
+  --country-selector-- {
+    color: var(--site-footer--color, $white);
```

Comment:
What is the motivation for using the footer variables here?

Reply:
As I understand, it could be to have the country selector color default to the general color of the site footer, but still allow changing it individually.

However it only has a small benefit to add `--site-footer--color` variable here, because you can also just put the same color into both names in the theme.

Perhaps we also don't want to have this color be separate for both. In that case we can just remove these rules and rely on CSS cascade of the `--site-footer` rules.

Reply:
> As I understand, it could be to have the country selector color default to the general color of the site footer, but still allow changing it individually.

yup

> you can also just put the same color into both names in the theme.

Right, I was adapting my code to themes with the intent of not modifying any theme config, because that was the easiest thing I found giving me a hint on what properties we need to be easily modifiable. Checking other sites, [nordic themes](https://www.greenpeace.org/sweden/) have a different color for footer and country selector, so that usage exists (but of course I don't know what they will do when the country selector moves to the footer).
Sooo, I think it makes sense like this, but removing the default fallback and adding country-selector variables to campaign themes will also work for me so I can also do it if you prefer ? and keeping those variables separated seems useful

Reply:
Yeah it makes sense as a temporary measure, in case there are some downstream themes that can't be updated.

For now [all themes we need to support](https://github.com/greenpeace/planet4-master-theme/tree/master/themes) are still inside the same repo, so we can adjust their config files and keep our code cleaner.

You can try using the experimental theme editor to generate the new config file. Should be straightforward if it uses colors that are already in the theme, you can select it from the suggestions. Unfortunately that only works for colors so with other things like font size it's a bit more manual. But might be easier to check some things than just using the JSON file.

I don't think any child theme depends on the CSS variables yet, they still use plain old CSS overrides. We still need to be sure we can offer them good usage steps so that we can handle new variables.